hw_config.h

```diff
@@ -805,6 +805,11 @@
 
 # define OSC_FREQ                       8
 
+// VBUS sense is connected to PC13 instead of PA9
+# define BOARD_USB_VBUS_SENSE_DISABLED
```

Comment:
For GPIO to be used the port's clock has to be enable in the RCC. This is done in RCC_AHBnENR  using a RCC_AHBnENR_GPIOaEN (where n:{1..N} and a:{A..J} N,J are dependent on IP of SOC.

For @bkueng case he got a free ride because the pin was on the same port. You will have to change all the sense code. Please use the negative define pattern found at the bottom of this file to keep the default at A.

Reply:
I didn't see a clean way to do it with defaults in the hw_config.h because then we start needing hardware specific defaults and it mixes the main_fx.c code into the hw_config.h a lot more. Let me know what you think of this solution?

Reply:
Nevermind, seems F4 and F7 share the default. Got confused at first with the GPIOxEN vs IOPxEN.